src/components/Modal/VaultStrategyModal/managedVaults/ManagedVaultsStrategyModalAllocations.tsx

```diff
@@ -0,0 +1,181 @@
+import { ArcElement, Chart as ChartJS, Colors, Legend, Tooltip } from "chart.js"
+import { FC } from "react"
+import { Doughnut } from "react-chartjs-2"
+import { Address } from "wagmi"
+
+import { formatPercentage } from "@/lib/helpers"
+
+import { AssetLogo } from "@/components/Asset"
+
+ChartJS.register(ArcElement, Tooltip, Legend, Colors)
+
+export const ManagedVaultsStrategyModalAllocations: FC = () => {
+  const addresses: Address[] = [
+    "0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9",
```

Comment:
This is just for demo purposes right? This would eventually come from the contract?

Reply:
All hardcode addresses and values are for the demo. Waiting for the BC before writing the hooks.